modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-vsphere_{context}"]
+= Upgrading from {product-title} 4.12 to 4.13 and later with vSphere in-tree PVs
+
+If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 to 4.13 or later, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} upgrade can occur and automatic CSI migration for vSphere only occurs:
+
+* For 4.13: only if you opt in.
+
+* For 4.14 and later: automatically by default.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} upgrade by:
+
+* Initiating the upgrade from a {product-title} 4.12 Z-stream.
+
+* Performing an administrator acknowledgment by running the following command:
++
+[source, terminal]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-1.26-api-removals-in-4.14":"true"}}' --type=merge
+----
+
+[IMPORTANT]
+====
+If you upgrade to {product-title} 4.13 or later without updating vSphere, and then, for 4.13, opt in to migration, known issues can occur. https://access.redhat.com/node/7011683[Before upgrading, carefully read this knowledge base article].
```

Comment:
Can we remove the _for 4.13_ here? It seems clear that is what you are talking about and it would make the sentence cleaner.

```suggestion
If you upgrade to {product-title} 4.13 or later without updating vSphere, and then opt in to migration, known issues can occur. https://access.redhat.com/node/7011683[Before upgrading, carefully read this knowledge base article].
```

Reply:
Only for 4.13 is opting in to migration available. Later version there is no opting in. I agree it's confusing, but it's accurate.